misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/utils/TypeExtractorUtil.java

```diff
@@ -396,8 +432,13 @@ public static String escapeIdentifier(String identifier) {
             if (identifier.equals("error")) {
                 identifier = "_error";
             } else {
-                identifier = identifier.replaceAll("([\\\\?!<>*\\-=^+()_{}|.$])", "$1");
+                identifier = identifier.replaceAll("([\\[\\]\\\\?!<>@#&~`*\\-=^+();:\\_{}\\s|.$])", "\\\\$1");
```

Comment:
Shall we consider using this regex as a constant, as I've noticed that this same pattern is duplicated in few other places as well. WDYT?